tests/src/Kernel/Entity/OgRoleTest.php

```diff
@@ -243,6 +243,76 @@ public function testRoleCreate() {
     }
   }
 
+  /**
+   * Tests searching roles by a given permission list.
+   */
+  public function testLoadRoleByPermissions() {
```

Comment:
This is actually testing code on the `OgRoleManager` so this test should be moved to `OgRoleManagerTest`.